src/ui/app/App.js

```diff
@@ -382,10 +381,10 @@ class App extends Component {
       : <LoginPage />;
 
     return (
-      <DndProvider backend={HTML5Backend}>
+      <DDProvider>
         <ToastsContainer />
         {!isReady ? null : readyDisplay}
-      </DndProvider>
+      </DDProvider>
```

Comment:
have you tested that this did not break existing DND for example for page list moving or widget dnd?

Reply:
@ichalagashvili yes this works because all of the DnD components/functions are now imported via @entando/dndtable. This was the reason why at the first place they moved the dndtable at the appbuilder because the provider came from app-builder while the dndtable came from an imported module. putting both provider and dndtable at the same source should make `react-dnd` work fine.